### ISSUE TITLE
[CRITICAL] [Advanced Approval Workflows] Implement `<ApprovalWorkflowSection />` component that displays a single

### ISSUE THREAD

**Issue by tgolen** [Expensify/App#45951]
Coming from: [#wave-control: advanced approval + workflows polish](https://docs.google.com/open?id=12X-hsNo9BBRcG5NSS1JzyjWD1w-iM0uciQJSD1i5mZc#bookmark=id.wo3hcc5upya6)

**Comment by JmillsExpensify** [comment]
@Guccio163 can you comment on this issue so I can assign you?

**Comment by Guccio163** [comment]
Yes, I just wanted to take it today, assign it to me and I'll probably finish it tomorrow!

**Comment by tgolen** [comment]
@VictoriaExpensify Can you please assist with paying @shubham1206agra for a C+ review of the PR?

**Comment by VictoriaExpensify** [comment]
Sorry, automation didn't work since the External label wasn't applied. Working on this now.

**Comment by VictoriaExpensify** [comment]
@shubham1206agra - I've sent you an [offer in Upwork](https://www.upwork.com/nx/wm/offer/103594812), can you please let me know when you've accepted it and I'll approve payment. Thanks!

**Comment by VictoriaExpensify** [comment]
**Payment Summary:**
**Contributor**: @Guccio163 is from an agency-contributor and not due payment
**Contributor+**: @shubham1206agra paid $250 via [Upwork](https://www.upwork.com/nx/wm/offer/103594812)


Thanks for your work on this issue!
